includes/abstracts/abstract-wc-legacy-product.php

```diff
@@ -661,4 +661,61 @@ public function get_matching_variation( $match_attributes = array() ) {
 		_deprecated_function( 'WC_Product::get_matching_variation', '2.7', 'wc_find_matching_product_variation' );
 		return wc_find_matching_product_variation( $this, $match_attributes );
 	}
+
+	/**
+	 * Returns whether or not we are showing dimensions on the product page.
+	 *  @deprecated 2.7.0 Unused.
```

Comment:
Hopefully I'm not just seeing things this time (:P), but it looks like there is an extra space in front.